openadapt/models.py

```diff
@@ -526,7 +545,16 @@ def get_active_window_event(
         Returns:
             (WindowEvent) the active window event.
         """
-        return WindowEvent(**window.get_active_window_data(include_window_data))
+        window_event_data = window.get_active_window_data(include_window_data)
+
+        a11y_event_data = window_event_data.get("state", {}).copy()
+        window_event_data.pop("state", None)
+        a11y_event_handle = window_event_data.get("handle")
+        a11y_event = A11yEvent(data=a11y_event_data, handle=a11y_event_handle)
```

Comment:
I think we only want to do this conditionally, e.g. with an argument `include_a11y_data=config.READ_A11_DATA` or similar. What do you think?

Reply:
But then we won't get `meta` if `include_a11y_data` is set to `False` as `meta` is in `state` and we removed `state` col from `window_event`